osu.Game.Rulesets.Taiko/Mods/TaikoModHidden.cs

```diff
@@ -28,7 +29,13 @@ public class TaikoModHidden : ModHidden, IApplicableToDrawableRuleset<TaikoHitOb
         /// How long hitobjects take to fade out, in terms of the scrolling length.
         /// Range: [0, 1]
         /// </summary>
-        private const float fade_out_duration = 0.375f;
+        public readonly Bindable<float> FadeOutDuration = new Bindable<float>(0.375f);
```

Comment:
For safety you can use `BindableFloat` and then set `MinValue` and `MaxValue` to the intended range (for this and the bindable below).

Reply:
Makes sense, applied in 8000ae6c54d4b10e7c37b7bc74cf1fba4c449411

As a side note, IBindable.cs in osu-framework seems to have a potential minor typo at line 18 and 91:
![image](https://user-images.githubusercontent.com/5846322/183444747-bdf86f01-07e7-4e7c-97f3-0e7784cb2a05.png)